backend/tarsy/hooks/typed_dashboard_hooks.py

```
@@ -39,21 +39,19 @@ async def execute(self, interaction: LLMInteraction) -> None:
             interaction: Unified LLM interaction data
         """
         try:
-            # Create dashboard update from typed interaction
+            # Create dashboard update with complete conversation object
             update_data = {
                 "type": "llm_interaction",
                 "session_id": interaction.session_id,
-                "request_id": interaction.request_id,
+                "request_id": interaction.interaction_id,  # Use interaction_id instead of removed request_id
```

Ruff (0.12.2)

Line too long (109 > 88)

(E501)